Model/Processor/ImportProcessor.php

```diff
@@ -85,8 +85,11 @@ public function __construct(
     public function process()
     {
         $files = $this->finder->find();
-        if (0 === count($files)) {
+        if (0 === count($files) && false === $this->getInput()->getOption('allow-empty-directories')) {
             throw new \InvalidArgumentException('No files found for format: *.' . $this->getFormat());
+        } else {
+            $this->getOutput()->writeln('No files found for format: *.' . $this->getFormat());
```

Comment:
I would like to leave the output, otherwise the import could look successful, no matter if some values have been processed or not. If you don't know the module well at the start you don't know that there will be an output of the imported values and it looks like this:

```
/var/www/html$ bin/magento config:data:import -e .config/store stage

  Start Import

  Clear cache
```